src/app/pages/license/license.hbs

```diff
@@ -0,0 +1,206 @@
+            </p>
+
+            <h3>Can people make money off of my work?</h3>
+            <p>
+                It is possible, but to be effective at all the license we use must allow
+                some "commercial" uses. Consider the situation in which a printing service,
+                such as FedEx Kinko's℠, offers to print your work so that your students
+                may have access to hard copies. Without a license that allows commercial
+                use, the service could not charge a fee to recover their printing, binding,
+                and handling costs.
+            </p>
+
+            <h3>That's fine, but what if some huge publisher decides to publish my work?</h3>
+            <p>
+                If a publisher wants to include your module in a book of collected materials,
+                they have that right according to the license, but remember that they
+                are required to list you as the author and copyright holder. Our experience
+                also suggests that most reputable publishers would want to re-negotiate
+                a new license with you in that case. Furthermore, if you seriously expect
+                to receive royalties from what you write, our repository is probably
+                not the right place for the work -- though it might be the right place
+                to experiment with an early version, or an alternative version.
+            </p>
+
+            <h3>Does the Creative Commons Attribution license affect fair use rights?</h3>
+            <p>
+                No. The license includes the statement: "Nothing in this license is intended
+                to reduce, limit, or restrict any rights arising from fair use, first
+                sale, or other limitations on the exclusive rights of the copyright owner
+                under copyright law or other applicable laws." Fair use, the first sale
+                doctrine, and other such limitations apply whether a copyright holder
+                consents to them or not.
+            </p>
+
+            <h3>What must I do if I want to include a portion of an author’s work in the
+            repository in a work I am creating outside of the repository?</h3>
+            <p>
+                You must give attribution to the portion of the original author’s work
+                that you include outside of our repository. This attribution is as simple
+                as clearly naming the original author and identifying the portion of
+                his or her work you are using. In addition you must retain the copyright
+                notice and provide the URI of the original work.
+            </p>
+
+            <h3>What must I do if I want to include another author’s module in a collection
+            that I am building?</h3>
+            <p>
+                Simply add the module to your collection with the Collection Composer tool.
```

Comment:
This should be: "Simply add the page to your book." Remove the reference to the collection composer.